metadata-ingestion/tests/integration/great-expectations/ge_mcps_golden.json

```diff
@@ -1,782 +1,1929 @@
 [
 {
-    "auditHeader": null,
     "entityType": "assertion",
     "entityUrn": "urn:li:assertion:358c683782c93c2fc2bd4bdd4fdb0153",
-    "entityKeyAspect": null,
     "changeType": "UPSERT",
     "aspectName": "assertionInfo",
     "aspect": {
-        "value": "{\"customProperties\": {\"expectation_suite_name\": \"test_suite\"}, \"type\": \"DATASET\", \"datasetAssertion\": {\"dataset\": \"urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo1,PROD)\", \"scope\": \"DATASET_COLUMN\", \"fields\": [\"urn:li:schemaField:(urn:li:dataset:(urn:li:dataPlatform:postgres,postgres1.postgres.public.foo1,PROD),score)\"], \"aggregation\": \"IDENTITY\", \"operator\": \"BETWEEN\", \"parameters\": {\"maxValue\": {\"value\": \"8\", \"type\": \"UNKNOWN\"}, \"minValue\": {\"value\": \"0\", \"type\": \"UNKNOWN\"}}, \"nativeType\": \"expect_column_values_to_be_between\", \"nativeParameters\": {\"column\": \"score\", \"max_value\": \"8\", \"min_value\": \"0\"}}}",
-        "contentType": "application/json"
-    },
-    "systemMetadata": null
+        "json": {
```

Comment:
These changes are rewriting mcp aspect values in expanded format for readability and does not include any changes specific to this PR.

Reply:
yup sounds good